Product.Data/Item.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Product.Schema.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Product.Data
+{
+    public class Item
+    {
+        private readonly ApplicationDbContext dbContext;
+
+        public Item(ApplicationDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public void AddItem(Schema.Models.Item item)
```

Comment:
Why you need to have fully qualified name here? Is there any conflict

Reply:
yes, as the Product.Data also contain same name 'Item' and 'ItemObjects' as Schema so I need to specify the full name.